locales-loader.js

```diff
@@ -0,0 +1,27 @@
+const buildConfig = require('./server/config.app');
+
+function localesLoader(source) {
+  if (buildConfig.TEST) {
+    return source;
+  }
+
+  const localesParams = JSON.parse(source);
+  for (const locale in localesParams.resources) {
+    if (Object.prototype.hasOwnProperty.call(localesParams.resources, locale)) {
+      const main = require(`./locales/${locale}/translation.json`);
+      const params = require(`./locales/${locale}/parameter.json`);
+      localesParams.resources[locale].main = main;
+      localesParams.resources[locale].params = params;
+    }
+  }
+
+  // FIXME: Crowdin translations should only be available
+  // when crowdin is active
```

Comment:
should we use the CROWDIN envt var here ?

Reply:
We will at terms.
But not for now, it is a subject for another pull request.
Will be done in https://diabeloop.atlassian.net/browse/YLP-106